code/modules/mob/living/carbon/examine.dm

```diff
@@ -198,38 +198,12 @@
 	if(reagents.has_reagent(/datum/reagent/teslium, needs_metabolizing = TRUE))
 		. += span_smallnoticeital("[t_He] [t_is] emitting a gentle blue glow!") // this should be signalized
 
+	var/mob/living/living_user = user
+	SEND_SIGNAL(living_user, COMSIG_EMPATHY_EXAMINE, src, .) // Empathy text comes before the other stuff and requires different arguments
```

Comment:
Signals are typically supposed to be generic events, and naming it something uber specific like `COMSIG_EMPATHY_EXAMINE` goes against that. 

Is it possible to simply use `COMSIG_ATOM_EXAMINE`? I know it'll change the order which it appends the empath messages, but typically I would just bite the bullet and let it fall to the bottom. 

If you are set on keeping it in its current order maybe we can name it something like `COMSIG_CARBON_MID_EXAMINE`.